src/packages/cli/src/Generate.ts

```diff
@@ -98,7 +96,7 @@ export class Generate implements Command {
       '--watch': Boolean,
       '--schema': String,
       // Only used for checkpoint information
-      '--postinstall': Boolean,
+      '--postinstall': String,
```

Comment:
@Weakky can you give me a tip about how to send the string value to telemetry? It looks like `--postinstall` and `--telemetry-information` aren't used at all right now?

Reply:
Ok, team has explained to me there is nothing more to do here actually.